build/stf-run-ci/tasks/deploy_stf.yml

```diff
@@ -34,15 +34,6 @@
           events:
             elasticsearch:
               enabled: {{ __service_telemetry_events_enabled }}
```

Comment:
In my last PR, Leif suggested I make the defaults match exactly what we currently deploy so that migrations will be easy. Since I just copied the template from the operator to the CI code, a side effect is that we don't need to set any values for testing purposes, just enable it and the forwarding is correctly configured.